src/DataSources/Clients.js

```diff
@@ -20,13 +20,21 @@ class ClientDataSource {
     }
 
     if (await config.get('api')) {
-      this.clients.apiClient = Promise.promisifyAll(require('classy-api-client')({
-        clientId: await config.get('APIV2_CLIENT_ID'),
-        clientSecret: await config.get('APIV2_CLIENT_SECRET'),
-        timeout: await config.get('api.timeout'),
-        oauthUrl: await config.get('api.oauthUrl'),
-        apiUrl: await config.get('api.apiUrl')
-      }));
+      if (await config.get('api.clientType') === 'classy-node') {
```

Comment:
*Still Researching

We should include the internal classy node to have access to all APIv2 methods

https://bitbucket.org/stayclassy/classy-node-internal-extension/src/master/

Reply:
all endpoints added
https://bitbucket.org/stayclassy/classy-node-internal-extension/src/master/src/resources.json

Reply:
I'm not sure we can do that and still have this be an external npm-hosted project.  Maybe it's time to make pieces of this non-external?